classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
```

Comment:
- Please, kindly also add the student to the accessor so that When adding a student to a classroom it also sets the classroom for the student.